aptos-move/framework/aptos-token-objects/sources/collection.move

```diff
@@ -119,18 +119,35 @@ module aptos_token_objects::collection {
         token: address,
     }
 
+    struct MintEvent has drop, store {
+        index: u64,
+        token: address,
+    }
+
     #[event]
-    struct ConcurrentBurnEvent has drop, store {
+    struct BurnEventV2 has drop, store {
```

Comment:
Can we avoid `V2` and `Event` in the naming since `#[event]` already means event?

Can we call it `Burn`?